manual/development-process/access-integration-db.md

```diff
@@ -0,0 +1,78 @@
+#Accessing the Integration database
+
+Connecting to the Integration database must be done through an AWS Bastion host, below are the steps detailing how to do this
+
+##1. Downloading and installing the Session Manager plugin
+First you need to download and install the AWS Session Manager plugin
+
+   * [Instructions for Mac (Step 1 to 3)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-macos)
+   * [Instructions for Linux (Step 1 to 2)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-linux)
+   * [Instructions for Ubuntu Server (Step 1 to 2)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-debian)
```

Comment:
When would you want to to run these instructions on Ubuntu Server? Is that different to the Ubuntu Desktop Linux  that some of the team are running?

Reply:
The instructions are the same for ubuntu desktop. What AWS mean by Linux instructions are Amazon Linux instructions, the ubuntu ones use a .deb package. Maybe just take the word Server out and it'll be fine.